include/mbedtls/build_info.h

```diff
@@ -61,8 +61,9 @@
 
 #if defined(MBEDTLS_CONFIG_VERSION) && ( \
     MBEDTLS_CONFIG_VERSION < 0x03000000 || \
-    MBEDTLS_CONFIG_VERSION > MBEDTLS_VERSION_NUMBER )
-#error "Invalid config version, defined value of MBEDTLS_CONFIG_VERSION is unsupported"
+                             MBEDTLS_CONFIG_VERSION > MBEDTLS_VERSION_NUMBER)
```

Comment:
This line's indentation doesn't make sense to me. (But we may just have to live with it, depending on what went through uncrustify's mind here — other similar places look fine.)

Reply:
I'm unsure exactly why it is this way but I expect it's related to having a boolean expressions indented further:
```c
long_condition_a &&
long_condition_b &&
    (long_condition_c ||
    long_condition_d)
```